gatsby-theme-marketing/static/specs-gen.json

```diff
@@ -1 +1 @@
ostgresql\n        uri: 'postgresql://user:password@hostname:5432/dbname?sslmode=require'\n  analyzers:\n    - postgres:\n        checkName: Must be PostgreSQL 10.x or later\n        collectorName: postgresql\n        outcomes:\n          - fail:\n              when: connected == false\n              message: Cannot connect to PostgreSQL server\n          - fail:\n              when: version < 10.x\n              message: The PostgreSQL server must be at least version 10\n          - pass:\n              message: The PostgreSQL server is ready\n","supportSpecYaml":"apiVersion: troubleshoot.sh/v1beta2\nkind: SupportBundle\nmetadata:\n  name: supported-postgres-version\nspec:\n  collectors:\n    - postgres:\n        collectorName: postgresql\n        uri: 'postgresql://user:password@hostname:5432/dbname?sslmode=require'\n  analyzers:\n    - postgres:\n        checkName: Must be PostgreSQL 10.x or later\n        collectorName: postgresql\n        outcomes:\n          - fail:\n              when: connected == false\n              message: Cannot connect to PostgreSQL server\n          - fail:\n              when: version < 10.x\n              message: The PostgreSQL server must be at least version 10\n          - pass:\n              message: The PostgreSQL server is ready\n"},{"slug":"supported-redis-version","category":"database","title":"Redis Version Check","description":"Verify that a Redis cluster is running a compatible version","iconUri":"/category-icons/redis.svg","contributors":[{"name":"Marc Campbell","avatarUri":"https://avatars3.githubusercontent.com/u/173451?s=460&v=4"}],"tags":["database","redis"],"preflightSpecYaml":"apiVersion: troubleshoot.sh/v1beta2\nkind: Preflight\nmetadata:\n  name: supported-redis-version\nspec:\n  collectors:\n    - redis:\n        collectorName: redis\n        uri: 'rediss://default:password@hostname:6379'\n  analyzers:\n    - redis:\n        checkName: Must be redis 5.x or later\n        collectorName: redis\n        outcomes:\n          - fail:\n              when: connected == false\n              message: Cannot connect to redis server\n          - fail:\n              when: version < 5.0.0\n              message: The redis server must be at least version 5\n          - pass:\n              message: The redis connection checks out\n","supportSpecYaml":"apiVersion: troubleshoot.sh/v1beta2\nkind: SupportBundle\nmetadata:\n  name: supported-redis-version\nspec:\n  collectors:\n    - redis:\n        collectorName: redis\n        uri: 'rediss://default:password@hostname:6379'\n  analyzers:\n    - redis:\n        checkName: Must be redis 5.x or later\n        collectorName: redis\n        outcomes:\n          - fail:\n              when: connected == false\n              message: Cannot connect to redis server\n          - fail:\n              when: version < 5.0.0\n              message: The redis server must be at least version 5\n          - pass:\n              message: The redis connection checks out\n"}]}
```

Comment:
I don't think it'll do any harm but we shouldn't commit this file. Probably should add it to the gitignore